test/js/pageobjects/DocumentUploadConfirmation.js

```diff
@@ -0,0 +1,14 @@
+import Base from './BasePage.js'
+
+class DocumentUploadConfirmation extends Base{
+
+  get makeSureClearDetailsMessage() { return this.$('.onfido-sdk-ui-PageTitle-titleWrapper > div:nth-child(2)')}
```

Comment:
this selector, with children, will likely be flaky

Reply:
I haven't noticed any flakiness so far with this selector. This is the only one from the ones I have managed to create and worked also.

Reply:
It looks like the PageTitle component's Subtitle area that this is testing for does not have any class assigned 😞